pkg/skaffold/build/local.go

```diff
@@ -128,3 +133,52 @@ func (l *LocalBuilder) Build(ctx context.Context, out io.Writer, tagger tag.Tagg
 
 	return res, nil
 }
+
+func (l *LocalBuilder) buildBazel(ctx context.Context, out io.Writer, a *config.Artifact) (string, error) {
+	cmd := exec.Command("bazel", "build", a.BazelArtifact.BuildTarget)
+	cmd.Stdout = out
+	cmd.Stderr = out
+	if err := cmd.Run(); err != nil {
+		return "", errors.Wrap(err, "running command")
+	}
+	//TODO(r2d4): strip off leading //:, bad
+	tarPath := strings.TrimPrefix(a.BazelArtifact.BuildTarget, "//:")
+	//TODO(r2d4): strip off trailing .tar, even worse
+	imageTag := strings.TrimSuffix(tarPath, ".tar")
+	imageTar, err := os.Open(fmt.Sprintf("bazel-bin/%s", tarPath))
```

Comment:
You might want to use `filepath.Join("bazel-bin", tarPath)`

Reply:
done